packages/webview_flutter/webview_flutter_android/pubspec.yaml

```diff
@@ -29,4 +29,4 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   mockito: ^5.3.2
-  pigeon: ^4.2.14
+  pigeon: ^9.0.4
```

Comment:
Since you are updating the pigeon version, could you try and regenerate the mocks as well. I tried raising the pigeon version too, but it seems the new analyzer was causing `mockito` to generate bad mocks. I want to see if you run into the same problem or if it was specific to me.

Reply:
It's not just you; the `Future<Offset>` and `Future<Size>` methods are wrong. I'm trying to make a reduced test case to file an issue, but a trivial class with a `Future<Offset>` works, so it seems like there's something more complicated going on.

Reply:
No luck so far. Maybe we should just file it with webview_flutter as-is, and hope someone who knows the internals better can figure out what's going on more easily that I can figure out what the minimal repro is.

How do you want to handle this in the meantime? Hold this PR, or manually fix it in the mocks with a TODO and issue link?

Reply:
Would it be possible just to revert the version bump to pigeon? Then this PR should be good.

Reply:
+1 to submitting with webview_flutter as-is. I think we can just give them the `android_webview_test.dart` with the `android_webview.dart` and say that the mockito outputs are incorrect. I can also handle making the issue if you would like.

Reply:
I filed https://github.com/dart-lang/mockito/issues/612, it turns out it's a dependency issue that doesn't reproduce if you clean or upgrade.

Reply:
Thanks for filing it!